src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__name {
+    display: inline-block;
```

Comment:
display: inline-block isn't necessary because your wrapper already place items as you need